app/controllers/public_pages_controller.rb

```diff
@@ -22,10 +22,11 @@ def template_export
     @template = Template.live(params[:id])
     # covers authorization for this action.
     # Pundit dosent support passing objects into scoped policies
+    skip_authorization
```

Comment:
prefer to just add this to the `verify_authorized` exception list above since we're already making exceptions.